codec-agnostic-rtp-payload-format.md

```diff
@@ -62,6 +62,39 @@ The spatial layers are sent in ascending order, with the same RTP timestamp, and
 If the application wants to encrypt content, the application will split media content into frames before encryption.
 Each frame will be encrypted independently, using {{SFrame}} for instance, and transmitted by the packetizer.
 
+Payload multiplexing
+=======================
+
+In order to reduce the number of payload type codes on the SDP exchange, a single payload type code for the generic packetization will be used for each media type. That requires to also carry the original payload type code for the negotiated media format that the frame belongs to.
+
+The associated payload type will be sent in a header extension. The payload of associated payload headerr extension element can be encoded using either the one-byte or two-byte header defined in [[RFC5285]].
+
+Figures 1 and 2 show sample encoding with each of these header formats.
+
+```
+                    0                   1
+                    0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5
+                   +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+                   |  ID   | len=0 |S|     APT     |
+                   +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+```
+Figure 1: Sample Associated Payload Type Encoding Using the One-Byte Header Format
+
+```
+      0                   1                   2                   3
+      0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
+     +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+     |      ID       |     len=1     |S|     APT     |    0 (pad)    |
+     +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+```
+Figure 2: Sample Associated Payload Type Encoding Using the Two-Byte Header Format
+
+The APT value is the value is the payload type code of the associated format passed to the rtp generic packetizer before any transformation is applied. The S bit indicates if the media stream can be forwarded safely starting from this RTP packet. Typically, it will be set to 1 on the first rtp packet of an I-Frame video frame and in all rtp audio packets.
+
+Receivers MUST be ready to receive RTP packets with different associated payload types in the same way it would receive different payload type codes on the RTP packets.
```

Comment:
I am not against the S bit, but it seems like something that is not related to the scope of this header extension. For instance it does not really make sense for audio codecs.
As a first step, I would simplify this header extension by removing the S bit.
Or write the possibility to add a S bit in this header extension as a note.

Reply:
The PT is only 7 bits, so there is a free bit we could use for free. The S bit is needed for video when no SVC (i.e. no Dependency Descriptor)  is used. That would allow to reduce the overhead a bit, but I agree it can be a bit weird.